config/create_fortran_test.py

```diff
@@ -0,0 +1,138 @@
+#
+# Use this script to create and initial <exercise>_test.f90 file 
+# which can be used as a skeleton for your test. Typically, you will
+# have to replace 'response' in the generated file with the right 
+# function call. 
+#
+# Also note that fortran has issues with special characters such as \n and \t 
+# so take special care handling these.
+#
+# 
+# Prerequsites
+# - Working cmake and fortran compiler
+# - Python3.x (it may work with Python2 but I have not tested)
+# - latest version of https://github.com/exercism/problem-specifications.git
```

Comment:
Some tracks add the problem specs repo as a [git submodule](https://git-scm.com/book/en/v2/Git-Tools-Submodules), so that it's easier to keep track of the version that was used to generate tests. A git submodule is essentially a reference to another git repo at a certain commit.

I think it's worth considering doing this to make it easier to keep the tests up-to-date if the problem specs change. I can go into further detail on Slack if you want.

Reply:
Some tracks require that the problem-specification repository be cloned at the same level as the language track.

So:

```plain
.                                                                                                                                               
├── automated-mentoring-support                                                                                                                 
├── bash                                                                                                                                        
├── fortran
├── haskell
├── problem-specifications
├── python
├── ruby
├── ruby-analyzer
├── vimscript
├── website
└── website-copy
```

Reply:
hmm, here I am a bit out of my knowledge of git to do this. Seems overkill to me to create a new git sub-repo for a single file(?) But I leave this to you guys to decide.

Reply:
A submodule isn't a new repo, it's a reference/link to a different repo at a certain commit. In the filesystem it's just a normal directory, but it allows you to refer to a specific version of that repo.

In this case the advantage of adding problem-specs as a submodule is that you could `pull` the latest version of it and update the exercises to that version. 
If problem-specs is updated after that, it's easy to find out what changed since the last time the exercises were updated, which can potentially make it easier to keep them up-to-date. The C# track does it, you can see what it looks like here: https://github.com/exercism/csharp/.

There are other ways to keep track of the changes to problem-specs, so it depends on your preferred workflow and is mostly a matter of preference.

I brought it up because I personally like that approach but perhaps it's best to ignore it for now and perhaps revisit it later once you have more experience with git and know what works better for you.